content/docs/cml/index.md

```diff
@@ -4,7 +4,7 @@ Continuous Machine Learning (CML) is an open-source library for implementing
 continuous integration & delivery (CI/CD) in machine learning projects. Use it
 to automate parts of your development workflow, including model training and
 evaluation, comparing ML experiments across your project history, and monitoring
-changing datasets.
+changing datasets. Website of the project https://cml.dev
```

Comment:
make `Continuous Machine Learning (CML)` at the top a link?

Reply:
Good idea - made CML as a link. I didn't remove the last sentence - it is better to make it explicit.